### PR TITLE
Feature branch for project

### PR DESCRIPTION
**Pull request by Othniel2471**
Added the link for the stylesheet in the index file:  created a stylesheet and linked it with the index.html file u
Added the node_modules folder to the gitignore file:  added the files I didn't want to push into my repo, into the gitignore file which was the node_module folder, and the test.md file.
 Added a readme file: created a READme file to give a good description of the project.

### CODE REVIEW

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Othniel2471 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966


## Highlights
✔Good use of github workflows
✔All linter checks are green 
✔Add requested files 

## Required Changes ♻️
Great job @Othniel2471  only small changes to do before getting the approval 

- [x] README: well done by using microverse template 💪 , but we need to keep all sections ony FAQ is optional and is don't have content for a section you can say coming soon or working on  

- [ ] PR description: kindly always use a title that describes the feature added and describe with more details what was added removed changed or optimized in the summary 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Othniel2471** [comment]
@souad988 
Thank you for the feedback, 
sorry please i am not really clear on the PR description you mentioned,
and also the changes i made in my readme file is in my features branch, i don't know if that is okay

File: READme.md [hunk 1]
```diff
@@ -37,6 +103,35 @@ Omaidu Othniel
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+coming soon!!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (usorfaitheloho) [inline READme.md:168]: You did well in adding back all sections as requested by the previous reviewer, however, kindly tailor the details in all sections to be in alignment with your project 😄 .. for instance, the `setup` section doesn't include this repo in the details, please add tools to be installed to run the project successfully, add future features and acknowledge people who made this project successful.

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
Hi @Othniel2471 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Added descriptions to the PR summary
- Good use of HTML and CSS syntax


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Eternalgratis — APPROVED** [review]
# Status: Approved  🎉
Hi @Othniel2471 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Good luck! 🎉 
Developer-512px-10 

## Highlights 
- README file is descriptive ✔
- Linter result is green ✔
- Github flow was correctly used ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**